tests/beaker_tests/cisco_portchannel_global/test_portchannel_global.rb

```diff
@@ -213,6 +213,35 @@
   },
 }
 
+tests['default_properties_no_hash'] = {
+  title_pattern:  'default',
+  manifest_props: "
+    bundle_hash                  => 'default',
+    bundle_select                => 'default',
+    rotate                       => 'default',
+  ",
+  code:           [0, 2],
```

Comment:
If the system is already in default, the test is failing because no changes are done. That is why 0,2 is added for all default tests. 
I put it for many beaker tests this way:
for ex:
https://github.com/cisco/cisco-network-puppet-module/blob/release_1.3.0/tests/beaker_tests/cisco_itd_device_group/test_itd_device_group.rb#L43

If there is a better way, let me know but I was doing this for a long time now.